app/views/invitations/show.html.haml

```diff
@@ -63,23 +63,18 @@
 .py-4.py-lg-5.bg-light
   .container
     .row
-      .col-12.col-lg-6
-        %h3 Schedule
+      - if @event.venue.present?
+        = render partial: 'shared/venue', locals: { venue: @event.venue, address: @host_address}
+      - else
+        %p= "#{t('events.virtual')}."
```

Comment:
Is the string interpolation here because of the full stop (`.`)? Do we need that dot? Could it just be the following?
```suggestion
        %p= t('events.virtual')
```

Reply:
@gnclmorais yeah, we probably do not need that dot or we can move it into the translation string itself

Reply:
@gnclmorais I've moved the dot into the translation string